### DISCUSSION TITLE
ET Pro Ruleset Install?

### DISCUSSION THREAD

**Discussion by lonelyadmin** [Security-Onion-Solutions/securityonion#1901] [Unsupported Versions] [answered]
I just purchased ET Pro Rulesets for my sensors and I have no idea where to configure the new rules. I've setup pulledpork before on standalone NIDS installs (from source, homegrown). Do I need to edit each sensor individually? Place the info in some pillar conf on the manager? 

I'm afraid of breaking something while I blindly look around /opt/so, /nsm, and others looking for the rule update config.

Can someone share how to change from the default ET Open ruleset to ET Pro.

I running the latest 2.3 installed from iso. Running several search and sensor nodes with a single manager.

Thanks! Forgive my "greenness"!

**Comment by TOoSmOotH** [comment, marked as answer]
I just updated the docs here: https://docs.securityonion.net/en/2.3/rules.html?highlight=idstools

Modify the pillar and then run ruleupdate

**Reply by lonelyadmin** [reply]
I updated that file as shown in docs, but running so-rule-update seems to show the same download of `2020-11-12 23:09:05,298 - <INFO> - Checking https://rules.emergingthreats.net/open/suricata-5.0.0/emerging.rules.tar.gz.md5.
`

Do I also need to update the "urls:" part in the manager sls file?

**Reply by lonelyadmin** [reply]
Nevermind....it picked up the change. I forgot that you either have to be patient or force the update to the clients.  I'm new to the salt thing :)

Thanks!
